File: maximumAndMinimum.js
function solution(s) {
    let k = s.split(" ")[0];
    var answer = [k, k];
    s.split(" ").forEach((a) => {
        if (answer[0] > +a) {
            answer[0] = +a;
        }
        if (answer[1] < +a) {
            answer[1] = +a;
        }
    });
    return answer.join(" ");
}

File: maximumAndMinimum.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s) {
    s += ' ';
    string answer = "";
    string tmp = "";
    int min = 2147483647;
    int max = -2147483648;
    for (int i = 0; i <= s.size(); i++) {
        if (s[i] != ' ') {
            tmp += s[i];
        } else {
            int k = stoi(tmp);
            if (min > k) {
                min = k;
            }
            if (max < k) {
                max = k;
            }
            tmp.clear();
        }
    }
    answer += to_string(min);
    answer += ' ';
    answer += to_string(max);
    return answer;
}